### PR TITLE
Improve/logging

### PR DESCRIPTION
**Pull request by vadymmarkov**
@zenangst @RamonGilabert @onmyway133 

**New level names** 
```swift
public enum LogLevel {
  case None, Error, Info, Verbose
}
```

**Request logging**
- Method + URL when the level is `Info` or `Verbose`;
- Headers and parameters when the level is `Verbose`;

**Response logging**
- Status code

### CODE REVIEW

**Comment by vadymmarkov** [comment]
Anyone?

File: Sources/Logging.swift [hunk 1]
```diff
@@ -2,51 +2,111 @@ import Foundation
...
+  public func logRequest(request: Requestable, URLRequest: NSURLRequest) {
+    guard let URLString = URLRequest.URL?.absoluteString else {
+      return
+    }
+
+    guard level == .Info || level == .Verbose else {
+      return
+    }
+
+    print("🏄 MALIBU: Catching the wave...")
```
Comment (RamonGilabert) [inline Sources/Logging.swift]: Such wave!

**Comment by RamonGilabert** [comment]
Travis fails, though @vadymmarkov. Fuck it?

**Comment by vadymmarkov** [comment]
@RamonGilabert I think Travis needs to be fixed, it has never worked here.

**Comment by RamonGilabert** [comment]
I'll totally do 🍣 today.

**Comment by zenangst** [comment]
That Travis...

**Pull request merged**